### PR TITLE
Nits

### PR DESCRIPTION
**Pull request by martinthomson**

### CODE REVIEW

File: draft-ietf-sframe-enc.md [hunk 1]
```diff
@@ -810,21 +810,21 @@ Epoch 17 +--+-- index=33 --> KID = 0x211
   ...
 ~~~~~
 {: #mls-evolution title="An example sequence of KIDs for an MLS-based SFrame
-session.  We assume that the group has 64 members, S=6." }
+session (E=4; S=6 or 64 group members)" }
```
Comment (bifurcation) [inline draft-ietf-sframe-enc.md:813]:
```suggestion
session (E=4; S=6, allowing for 64 group members)" }
```

File: draft-ietf-sframe-enc.md [hunk 2]
```diff
@@ -855,9 +855,8 @@ frame encryptor and assigned an unique counter for each.
 
 In both temporal and spatial scalability, the SFU may choose to drop layers in
 order to match a certain bitrate or forward specific media sizes or frames per
-second. In order to support it, the sender MUST encode each spatial layer of a
-given picture in a different frame. That is, an RTP frame may contain more than
-one SFrame encrypted frame with an incrementing frame counter.
+second. In order to support the SFU selectively removing layers, the sender MUST
+encode each layer in a different frame.
```
Comment (bifurcation) [inline draft-ietf-sframe-enc.md:859]:
```suggestion
second. In order to support the SFU selectively removing layers, the sender MUST
encapsulate each layer in a different SFrame ciphertext.
```

File: package.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (bifurcation) [inline package.json:1]: Do we need this file?

Comment (martinthomson) [reply package.json:1]: Yes.  If you use aasvg, you need this so that it builds for people who don't have it installed globally, at a minimum.

**Review by bifurcation — APPROVED** [review]

**Pull request merged**